src/Microsoft.Azure.SignalR.Common/Auth/JsonExtensions.cs

```diff
@@ -0,0 +1,102 @@
+﻿/* Modified From https://github.com/AzureAD/azure-activedirectory-identitymodel-extensions-for-dotnet/blob/f3ddfe0dcb40ec5bde3e324b4246984753a0e121/src/System.IdentityModel.Tokens.Jwt/JsonExtensions.cs
+ * Comparing with the original code
+ *   Remove
+ *      - method `DeserializeJwtHeader`
+ *      - method `DeserializeJwtPayload`
+ *   Add
+ *      - method `DeserializeJwtData`
```

Comment:
don't find DeserializeJwtData, make the comment consistent with the code?

Reply:
This file has been removed now